features/setup.py

```diff
@@ -21,6 +21,7 @@
     'numpy',
     'pandas',
     'scikit-learn',
+    'holidays',
```

Comment:
need to version cap. The others don't appear version capped but they are since we use a global version cap logic across shared dependencies: https://github.com/awslabs/autogluon/blob/master/core/src/autogluon/core/_setup_utils.py

Probably `'holidays>=0.13,<0.14'`